web-bundle/src/main/java/com/graphhopper/resources/RouteResource.java

```diff
@@ -151,10 +148,10 @@ public Response doGet(
 
         GHResponse ghResponse = graphHopper.route(request);
 
-        float took = sw.stop().getSeconds();
+        long took = sw.stop().getNanos() / 1000;
         String infoStr = httpReq.getRemoteAddr() + " " + httpReq.getLocale() + " " + httpReq.getHeader("User-Agent");
         String logStr = httpReq.getQueryString() + " " + infoStr + " " + requestPoints + ", took:"
-                + took + ", " + algoStr + ", " + weighting + ", " + vehicleStr;
+                + took + "micros, algo: " + algoStr + ", profile: " + profileName + ", " + weightingVehicleLogStr;
```

Comment:
@easbar for better readability IMO we should use milliseconds: `+ took/1000f + "ms...`?

Reply:
Ok micros might still be a bit too optimistic ;) I changed to millisecond with one decimal place

Reply:
:+1: